core/domain/topic_jobs_one_off_test.py

```diff
@@ -135,8 +135,18 @@ def test_migration_job_converts_old_topic(self):
         topic = (
             topic_models.TopicModel.get(self.TOPIC_ID))
         self.assertEqual(topic.subtopic_schema_version, 1)
+        self.assertEqual(
+            topic.subtopics[0],
```

Comment:
Same note here -- why the difference between this check and the one on line 148?